pkg/server/debug/logspy.go

```diff
@@ -87,8 +87,8 @@ func (d durationAsString) String() string {
 }
 
 const (
-	logSpyDefaultDuration = durationAsString(10 * time.Second)
-	logSpyMaxDuration     = durationAsString(time.Minute)
+	logSpyDefaultDuration = durationAsString(time.Minute)
```

Comment:
I personally wouldn't change the default duration, given that there's no good way to cancel a `logspy`. On a few occasions now I've accidentally started one before realizing I used the wrong `grep` argument, but had to wait out the 10 seconds before I could start a new one. Having to wait a minute would have been much more annoying.

Reply:
The one-at-a-time limit is one reason we put an upper bound on the time. It would be annoying if someone came along and started a logspy with a duration of 100000 hours, locking you out of this feature. I think some upper bound (maybe an hour?) still makes sense until we have a way to cancel.

Reply:
Hmm but we have cancellation here and the incoming request should be cancelled when your browser goes away (this is http2, right?). There's a test that verifies cancellation works when the incoming request context aborts.
Anyway, I changed this to 5s.